test/sponsorship-paymaster/biconomy-verifying-paymaster-v2-specs.ts

```diff
@@ -259,8 +237,44 @@ describe("EntryPoint with VerifyingPaymaster Singleton", function () {
       );
       userOp.signature = signatureWithModuleAddress;
 
-      await entryPoint.handleOps([userOp], await offchainSigner.getAddress());
-      // gas used VPM V2  162081
+      const tx = await entryPoint.handleOps(
+        [userOp],
+        await offchainSigner.getAddress()
+      );
+      const receipt = await tx.wait();
+      console.log("gas used VPM V2 ", receipt.gasUsed.toString());
+      console.log("gas price ", receipt.effectiveGasPrice.toString());
+
+      const bundlerPaid = receipt.effectiveGasPrice.mul(receipt.gasUsed);
+      console.log("bundler paid ", bundlerPaid.toString());
+
+      const paymasterFundsAfter = await entryPoint.balanceOf(paymasterAddress);
+      const paymasterIdBalanceAfter =
+        await verifyingSingletonPaymaster.getBalance(fundingId);
+
+      const paymasterIdBalanceDiff = paymasterIdBalanceBefore.sub(
+        paymasterIdBalanceAfter
+      );
+      console.log("paymasterIdBalanceDiff ", paymasterIdBalanceDiff.toString());
+
+      const paymasterFundsDiff = paymasterFundsBefore.sub(paymasterFundsAfter);
+      console.log("paymasterFundsDiff ", paymasterFundsDiff.toString());
+
+      // Review
+      // 10/11 actual gas used
+      /* const paymasterIdBalanceDiffWithoutPremium = paymasterIdBalanceDiff
+        .mul(BigNumber.from(10))
+        .div(BigNumber.from(11));
+
+      console.log(
+        "paymasterIdBalanceDiffWithoutPremium ",
+        paymasterIdBalanceDiffWithoutPremium.toString()
+      ); */
```

Comment:
please remove

Reply:
I'd like to keep it if I need to present more accounting check in future.